driver-core/src/main/java/com/datastax/driver/core/ControlConnection.java

```diff
@@ -516,6 +516,8 @@ private static void updateInfo(Host host, Row row, Cluster.Manager cluster, bool
             String dseVersion = row.getString("dse_version");
             host.setDseVersion(dseVersion);
         }
+        host.setHostId(row.getUUID("host_id"));
```

Comment:
I was skeptical that these would always be present in the column metadata, but as far as I know these values are always present going back to 1.2, CI seems to agree with that.